packages/common/src/ui/components/steppers/VerticalStepper/VerticalStepper.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { FC } from 'react';
+import Box from '@mui/material/Box';
+import Stepper from '@mui/material/Stepper';
+import Step from '@mui/material/Step';
+import StepLabel from '@mui/material/StepLabel';
+import Typography from '@mui/material/Typography';
+import StepConnector, {
+  stepConnectorClasses,
+} from '@mui/material/StepConnector';
+import { styled } from '@mui/material';
+
+import { LocaleKey, useTranslation } from '../../../../intl';
+
+interface StepDefinition {
+  label: LocaleKey;
+  description: string;
+}
+
+interface StepperProps {
+  activeStep: number;
+  steps: StepDefinition[];
+}
+
+const StyledConnector = styled(StepConnector)(({ theme }) => ({
+  '&.MuiStepConnector-root': {
+    // This margin is an adjustment as the icon we use: the circle, is smaller
+    // than what is the default size, so this tries to centre align the connector by
+    // overriding the default.
+    marginLeft: '3px',
+  },
+  [`& .${stepConnectorClasses.line}`]: {
+    borderColor: theme.palette.secondary.light,
+
+    // The width is for the connector which is significantly thicker than the default.
+    // Additionally the height has been shrunk so that each connector hits the icons
+    // on the top and bottom sufficiently.
+    borderWidth: 6,
+    minHeight: 12,
+  },
+  [`&.${stepConnectorClasses.active}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      borderColor: theme.palette.secondary.dark,
+    },
+  },
+
+  [`&.${stepConnectorClasses.completed}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      borderColor: theme.palette.secondary.dark,
+    },
+  },
+}));
+
+const Circle = styled('div', {
```

Comment:
Because there is the circle with just a border, I went with just making this circle div rather than the 'icon'